vscode-dotnet-runtime-extension/src/test/functional/DotnetCoreAcquisitionExtension.test.ts

```diff
@@ -124,4 +132,21 @@ suite('DotnetCoreAcquisitionExtension End to End', function() {
       assert.exists(versionError);
     }
   }).timeout(2000);
+
+  test('Install Command Passes With Warning With No RequestingExtensionId', async () => {
+    const context: IDotnetAcquireContext = { version: '3.1' };
+    const result = await vscode.commands.executeCommand<IDotnetAcquireResult>('dotnet.acquire', context);
+    assert.exists(result);
+    assert.exists(result!.dotnetPath);
+    assert.include(result!.dotnetPath, context.version);
+    assert.include(mockDisplayWorker.warningMessage, 'Ignoring existing .NET paths');
+  }).timeout(40000);
+
+  test('Install Command With Path Config Defined', async () => {
+    const context: IDotnetAcquireContext = { version: '0.1', requestingExtensionId: 'alternative.extension' };
+    const result = await vscode.commands.executeCommand<IDotnetAcquireResult>('dotnet.acquire', context);
+    assert.exists(result);
+    assert.exists(result!.dotnetPath);
+    assert.equal(result!.dotnetPath, 'foo');

```

Comment:
You manual tested the real thing at least once? A mock in this level does not provide 100% confidence.

Reply:
Yes, I have tested manually